core/backend/src/BlobContainerService.ts

```diff
@@ -45,26 +45,21 @@ export namespace BlobContainer {
     iTwinId: Id64String;
     /** optionally, an iModelId within the iTwin. If present, container is deleted when the iModel is deleted. */
     iModelId?: Id64String;
-    /** optionally, an owner of the container. Owners always have administrator rights for the container. */
-    owner?: string;
   }
 
   /**
    * Metadata about the use of a container so that:
    *  - administrators can understand why a container exists for assigning RBAC permissions appropriately
    *  - usage reports can aggregate types of containers
    *  - applications can identify their containers
-   *  - applications can store important (to them) properties "on" their containers
    */
   export interface Metadata {
-    /** Human readable explanation of the information held in the container. This will be displayed in the administrator RBAC panel, and on usage reports. */
-    description: string;
-    /** a machine-readable string that describes the "format" of the data in this container (e.g. "CloudSqlite") */
-    format: string;
-    /** an identifier of the application that uses this this container */
-    application: string;
-    /** Additional properties stored on the container */
-    [propertyName: string]: string;
+    /** Human-readable name for the container. This will be displayed in the administrator RBAC panel, and on usage reports. Non-unique.*/
+    name: string;
```

Comment:
`name` is a bad name for this (pun intended). Maybe `rbacLabel` or something. Should it be required?

Reply:
I agree, will change name to rbacLabel. As for the requirement, I set it as required since I updated description to be optional. I think one of the two should be required for some sort of description, but I could perhaps set a default value for name in the client or just allow it all to be optional.

Reply:
What exactly is wrong with `name`?

Every other type of object we manage in RBAC (iTwins, iModels, etc.) just has a `name` property - why shouldn't storage containers?

I could see some cases where the name is often (or always) generated (e.g., "ViewStore for {iModel Name}"), in which case I would concede that the "name" is acting more like a label, but surely there are other uses where container would benefit from having a proper name.  And without requiring one, how would an end-user administrator possibly understand what they're granting access to?

Reply:
Because it will be confused with [container name](https://learn.microsoft.com/en-us/rest/api/storageservices/naming-and-referencing-containers--blobs--and-metadata#container-names). That’s what I thought it meant.

Reply:
I see - that makes sense...

How about just calling this `label` then?

Reply:
I like `label`. There’s no expectation of uniqueness.